### DISCUSSION TITLE
Suggestions for new features from Australia

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#19] [New Feature Request]
Dear all,

Writing in this discussion a list of possible new features and suggestions for Mukurtu 4. Apologies if some of the below have been mentioned already! The suggestions below come from the work undertaken with communities as part of the NSW Australian Mukurtu Hub, but also from my personal experience of using Mukurtu as my PhD project:

- Timeline, infographics and Tagcloud to enrich content directly in Mukurtu 
- More fonts and colours to choose from to customise the homepage
- More templates for both homepage and basic pages (this could include, for example, sticky headers for a more contemporary look and feel)
- pop-ups to be used on the Homepage and the browse pages to include acknowledgments of country and warnings (personally, this has been a big thing for me that I would have liked to use in different instances). I found it relevant in particular when users access pages of the site from a research engine and can therefore miss the homepage text. 
- integration with multilingual software, if need a Mukurtu site in double language (I imagine this can be done in drupal with customisation, but I wonder if there is a simple way to do it. The solution I found in my case was to link a Mukurtu and a Wordpress site, but it's not ideal)
- transcription tool integrated into the DHI (but I think this has been on the radar for some time), as with 3D objects
- better visualisation of transcriptions (now you need to click on the 'transcription' button to visualise the text)
- I love the 'subcollections' function and I think there are very important, but the feedback I had over time is that the design is not clear. Maybe could be a design similar to the 'collections' page but more customisable?
- Option for having the Mukurtu site completely private
- Face tagging (maybe a bit down the track!).

Thank you all!

Monica

**Comment by alexmerrill** [comment]
@MonicaGalassi   Thank for all this great feedback, could you break this up into individual or, at least grouped, requests.  As you say some of these, we already have requests for and adding a +1 to say the [3D Media objects request](https://github.com/MukurtuCMS/Mukurtu-CMS/discussions/10) that @michael-wynne-wsu seeded would be helpful in us tracking how many people are interested and make it easier for us to manage requests overall and turn them into individual action items.  Thanks!

**Comment by michael-wynne-wsu** [comment]
Locking this conversation since all items have been moved to individual topics.
